api/Insight_API_documentation.md

```diff
@@ -0,0 +1,1200 @@
+}
+```
+
+<br/>
+
+### /blocks/
+
+**URL:**  ```GET /blocks?limit=N&blockDate=YYYY-MM-DD```
+
```

Comment:
I'd say the part after `/blocks` is not part of the URL.  the remaining part of that is the parameters.   the example below makes it very clear.